s3/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "common_tags" {
 }
 
 variable "logging_bucket_policy" {
-  default = ""
+  default = "Additional logging bucket policy to be added to a default policy."
```

Comment:
Did you mean to put this as the description instead of the default?

Reply:
fixed, good spot